test/e2e/ha/command_test.go

```diff
@@ -338,15 +340,15 @@ func getHACommandCluster(namespace, nodeLabel, storagePath string) *contrail.Man
 
 	webui := &contrail.WebuiService{
 		ObjectMeta: contrail.ObjectMeta{
-			Name:      "webui",
+			Name:      "command-ha",
```

Comment:
shouldn't it be command-ha-webui?

Reply:
not really, actually I would name all of other resources the same, unfortunately other controllers requires unique names. In my opinion it is rather issue with other controllers that they need to have unique names.

Reply:
but it looks really inconsistent right now. I agree and I think we talked about that some time ago that the type in the resource name is unnecessary but right now we have some resources with command-ha names and the others with command-ha-type names :/

Reply:
Ok I will change that